explorer/idl/explorer_v2.idl

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+// To compile this file:
+// 1. Install the barrister translator (IDL -> JSON)
+// you need to be root (or use sudo)
+// pip install barrister
+
+// 2. Install barrister-go
+// go get github.com/coopernurse/barrister-go
+// go install github.com/coopernurse/barrister-go/idl2go
+
+// 3. barrister explorer_v2.idl | $GOPATH/bin/idl2go -i -p explorer_v2
+
+struct CoinStatistic {
+    height int
+    supply string
+    numActions int
+    aps int
+}
+
+struct BlockGenerator {
+    name string
+    address string
+}
+
+struct BlockView {
+    ID string
+    height int
+    timestamp int
+    numActions int
+    generateBy BlockGenerator
+    amount string
+    size int
+    txRoot string
+    deltaStateDigest string
+}
+
+struct Log {
+    address string
+    topics []string
+    data string
+    blockNumber int
+    txnHash string
```

Comment:
txnHash is the only place that uses the prefix `txn` - should we make it `txHash` and then keep the consistency?

Reply:
Agree. Let's make it txHash.